deps/rabbitmq_web_mqtt/test/command_SUITE.erl

```diff
@@ -0,0 +1,179 @@
+%% This Source Code Form is subject to the terms of the Mozilla Public
+%% License, v. 2.0. If a copy of the MPL was not distributed with this
+%% file, You can obtain one at https://mozilla.org/MPL/2.0/.
+%%
+%% Copyright (c) 2007-2024 Broadcom. All Rights Reserved. The term “Broadcom” refers to Broadcom Inc. and/or its subsidiaries. All rights reserved.
+
+
+-module(command_SUITE).
+-compile([export_all, nowarn_export_all]).
+
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("amqp_client/include/amqp_client.hrl").
+-include_lib("rabbitmq_mqtt/include/rabbit_mqtt.hrl").
+-import(rabbit_web_mqtt_test_util, [connect/3, connect/4]).
+
+-define(COMMAND, 'Elixir.RabbitMQ.CLI.Ctl.Commands.ListWebMqttConnectionsCommand').
+
+all() ->
+    [
+     {group, unit},
+     {group, v5}
+    ].
+
+groups() ->
+    [
+     {unit, [], [merge_defaults]},
+     {v5, [], [run,
+               user_property]}
+    ].
+
+suite() ->
+    [
+      {timetrap, {minutes, 10}}
+    ].
+
+init_per_suite(Config) ->
+    rabbit_ct_helpers:log_environment(),
+    Config1 = rabbit_ct_helpers:set_config(Config, [
+        {rmq_nodename_suffix, ?MODULE},
+        {rmq_extra_tcp_ports, [tcp_port_mqtt_extra,
+                               tcp_port_mqtt_tls_extra]},
+        {rmq_nodes_clustered, true},
+        {rmq_nodes_count, 3}
+      ]),
+    rabbit_ct_helpers:run_setup_steps(Config1,
+      rabbit_ct_broker_helpers:setup_steps() ++
+      rabbit_ct_client_helpers:setup_steps()).
+
+end_per_suite(Config) ->
+    rabbit_ct_helpers:run_teardown_steps(Config,
+      rabbit_ct_client_helpers:teardown_steps() ++
+      rabbit_ct_broker_helpers:teardown_steps()).
+
+init_per_group(unit, Config) ->
+    Config;
+init_per_group(Group, Config) ->
+    case rabbit_ct_helpers:is_mixed_versions() of
```

Comment:
Could you please check a real condition that the old version can't be used as part of this test?

`is_mixed_versions()` will still return true in 10 years when testing RabbitMQ 21.x against 22.x. This function should almost never be used.

Reply:
Fixed in https://github.com/rabbitmq/rabbitmq-server/pull/10825